### PR TITLE
Clean authority after removing PLAT-1897

### PR DESCRIPTION
**Pull request by krokhmaliuk-virtru**
https://virtru.atlassian.net/browse/PLAT-1897

### CODE REVIEW

File: src/containers/Authorities/Authorities.tsx [hunk 1]
```diff
@@ -22,6 +23,9 @@ const Authorities = () => {
         })
         await getAuthorities();
         toast.success(`Authority ${authority} deleted`);
+        AttributesFiltersStore.update(s => {
```
Comment (sievdokymov-virtru) [inline src/containers/Authorities/Authorities.tsx:26]: Is this reset applied on deletion _currently selected authority_ only? Or any authority?
(Think we shouldn’t reset active authority on the Attributes page if some other authority item was deleted)

**Review by ivanovSPvirtru — DISMISSED** [review]

File: src/containers/Authorities/Authorities.tsx (hunk 1 above)
Comment (obidenko) [inline src/containers/Authorities/Authorities.tsx:26]: Could we, also, define an interface to the "s" variable?

Comment (krokhmaliuk-virtru) [reply src/containers/Authorities/Authorities.tsx]: Updated

Comment (krokhmaliuk-virtru) [reply src/containers/Authorities/Authorities.tsx]: TS pulls fields from the state
Screenshot 2022-07-15 at 15 32 44

**Pull request merged**